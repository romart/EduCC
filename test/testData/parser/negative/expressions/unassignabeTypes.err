test/testData/parser/negative/expressions/unassignabeTypes.c:5:25: error: assigning to 'signed int' from incompatible type 'STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:26: error: assigning to 'signed int' from incompatible type 'STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:34: error: assigning to 'float' from incompatible type 'signed int[20]'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:35: error: assigning to 'float' from incompatible type 'STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:36: error: assigning to 'float' from incompatible type 'STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:37: error: assigning to 'float' from incompatible type '*signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:38: error: assigning to 'float' from incompatible type '*float'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:39: error: assigning to 'float' from incompatible type '*STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:40: error: assigning to 'float' from incompatible type '*STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:42: error: assigning to 'STRUCT S1' from incompatible type 'signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:43: error: assigning to 'STRUCT S1' from incompatible type 'float'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:44: error: assigning to 'STRUCT S1' from incompatible type 'signed int[20]'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:46: error: assigning to 'STRUCT S1' from incompatible type 'STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:47: error: assigning to 'STRUCT S1' from incompatible type '*signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:48: error: assigning to 'STRUCT S1' from incompatible type '*float'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:49: error: assigning to 'STRUCT S1' from incompatible type '*STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:50: error: assigning to 'STRUCT S1' from incompatible type '*STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:52: error: assigning to 'STRUCT S2' from incompatible type 'signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:53: error: assigning to 'STRUCT S2' from incompatible type 'float'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:54: error: assigning to 'STRUCT S2' from incompatible type 'signed int[20]'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:55: error: assigning to 'STRUCT S2' from incompatible type 'STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:57: error: assigning to 'STRUCT S2' from incompatible type '*signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:58: error: assigning to 'STRUCT S2' from incompatible type '*float'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:59: error: assigning to 'STRUCT S2' from incompatible type '*STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:60: error: assigning to 'STRUCT S2' from incompatible type '*STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:62: error: array type 'signed int[20]' is not assignable
test/testData/parser/negative/expressions/unassignabeTypes.c:7:63: error: array type 'signed int[20]' is not assignable
test/testData/parser/negative/expressions/unassignabeTypes.c:7:64: error: array type 'signed int[20]' is not assignable
test/testData/parser/negative/expressions/unassignabeTypes.c:7:65: error: array type 'signed int[20]' is not assignable
test/testData/parser/negative/expressions/unassignabeTypes.c:7:66: error: array type 'signed int[20]' is not assignable
test/testData/parser/negative/expressions/unassignabeTypes.c:7:67: error: array type 'signed int[20]' is not assignable
test/testData/parser/negative/expressions/unassignabeTypes.c:7:68: error: array type 'signed int[20]' is not assignable
test/testData/parser/negative/expressions/unassignabeTypes.c:7:69: error: array type 'signed int[20]' is not assignable
test/testData/parser/negative/expressions/unassignabeTypes.c:7:70: error: array type 'signed int[20]' is not assignable
test/testData/parser/negative/expressions/unassignabeTypes.c:6:73: error: assigning to '*signed int' from incompatible type 'float'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:75: error: assigning to '*signed int' from incompatible type 'STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:76: error: assigning to '*signed int' from incompatible type 'STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:83: error: assigning to '*float' from incompatible type 'float'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:85: error: assigning to '*float' from incompatible type 'STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:86: error: assigning to '*float' from incompatible type 'STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:93: error: assigning to '*STRUCT S1' from incompatible type 'float'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:95: error: assigning to '*STRUCT S1' from incompatible type 'STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:96: error: assigning to '*STRUCT S1' from incompatible type 'STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:103: error: assigning to '*STRUCT S2' from incompatible type 'float'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:105: error: assigning to '*STRUCT S2' from incompatible type 'STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:106: error: assigning to '*STRUCT S2' from incompatible type 'STRUCT S2'
