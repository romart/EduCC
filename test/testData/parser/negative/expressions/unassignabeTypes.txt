FILE test/testData/parser/negative/expressions/unassignabeTypes.c
  STRUCT S1
----
  STRUCT S2
----
  signed int g = 10
----
  float gf = 20.000000
----
  signed int[20] garr
----
  FUN signed int main 
  BEGIN
    signed int i
    float f
    STRUCT S1 s1
    STRUCT S2 s2
    signed int[20] arr
    *signed int pi
    *float fp
    *STRUCT S1 ps1
    *STRUCT S2 ps2
    i = g
    i = gf
    i = garr
    i = s1
    i = s2
    i = pi
    i = fp
    i = ps1
    i = ps2
    f = g
    f = gf
    f = garr
    f = s1
    f = s2
    f = pi
    f = fp
    f = ps1
    f = ps2
    s1 = g
    s1 = gf
    s1 = garr
    s1 = s1
    s1 = s2
    s1 = pi
    s1 = fp
    s1 = ps1
    s1 = ps2
    s2 = g
    s2 = gf
    s2 = garr
    s2 = s1
    s2 = s2
    s2 = pi
    s2 = fp
    s2 = ps1
    s2 = ps2
    arr = g
    arr = gf
    arr = garr
    arr = s1
    arr = s2
    arr = pi
    arr = fp
    arr = ps1
    arr = ps2
    pi = g
    pi = gf
    pi = garr
    pi = s1
    pi = s2
    pi = pi
    pi = fp
    pi = ps1
    pi = ps2
    fp = g
    fp = gf
    fp = garr
    fp = s1
    fp = s2
    fp = pi
    fp = fp
    fp = ps1
    fp = ps2
    ps1 = g
    ps1 = gf
    ps1 = garr
    ps1 = s1
    ps1 = s2
    ps1 = pi
    ps1 = fp
    ps1 = ps1
    ps1 = ps2
    ps2 = g
    ps2 = gf
    ps2 = garr
    ps2 = s1
    ps2 = s2
    ps2 = pi
    ps2 = fp
    ps2 = ps1
    ps2 = ps2
    RETURN 0    
  END