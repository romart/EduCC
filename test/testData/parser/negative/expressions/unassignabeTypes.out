test/testData/parser/negative/expressions/unassignabeTypes.c:5:24: warning: incompatible integer to pointer conversion assigning to 'signed int' from 'signed int[20]'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:27: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:28: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*float'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:29: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:5:30: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:72: warning: incompatible integer to pointer conversion assigning to '*signed int' from 'signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:78: warning: incompatible pointer types assigning to '*signed int' from '*float'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:79: warning: incompatible pointer types assigning to '*signed int' from '*STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:80: warning: incompatible pointer types assigning to '*signed int' from '*STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:82: warning: incompatible integer to pointer conversion assigning to '*float' from 'signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:84: warning: incompatible pointer types assigning to '*float' from 'signed int[20]'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:87: warning: incompatible pointer types assigning to '*float' from '*signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:89: warning: incompatible pointer types assigning to '*float' from '*STRUCT S1'
test/testData/parser/negative/expressions/unassignabeTypes.c:6:90: warning: incompatible pointer types assigning to '*float' from '*STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:92: warning: incompatible integer to pointer conversion assigning to '*STRUCT S1' from 'signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:94: warning: incompatible pointer types assigning to '*STRUCT S1' from 'signed int[20]'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:97: warning: incompatible pointer types assigning to '*STRUCT S1' from '*signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:98: warning: incompatible pointer types assigning to '*STRUCT S1' from '*float'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:100: warning: incompatible pointer types assigning to '*STRUCT S1' from '*STRUCT S2'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:102: warning: incompatible integer to pointer conversion assigning to '*STRUCT S2' from 'signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:104: warning: incompatible pointer types assigning to '*STRUCT S2' from 'signed int[20]'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:107: warning: incompatible pointer types assigning to '*STRUCT S2' from '*signed int'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:108: warning: incompatible pointer types assigning to '*STRUCT S2' from '*float'
test/testData/parser/negative/expressions/unassignabeTypes.c:7:109: warning: incompatible pointer types assigning to '*STRUCT S2' from '*STRUCT S1'
